### PR TITLE
Update exitcontroller.java

### PR DESCRIPTION
**Pull request by iota333**
i have updated the code for ExitController.java please review it and let me know for any further changes

### CODE REVIEW

File: src/bcccp/carpark/exit/ExitController.java [hunk 1]
```diff
@@ -21,40 +21,122 @@
 	private long exitTime;
 	private String seasonTicketId = null;
 	
-	
+	/**
+ * This file used to Exit the car from the car park. This file has setState method to set the state of the car park, 
+ * log method used to create the log file, ticketInserted method, ticketTaken method, carEventDetected used when the car exit,
+ * change the state accordingly.
```
Comment (engrhxn) [inline src/bcccp/carpark/exit/ExitController.java:27]: Good Explanations.

File: src/bcccp/carpark/exit/ExitController.java [hunk 2]
```diff
@@ -21,40 +21,122 @@
...
+	 * 
+	 * @param message
+	 */
+	private void log(String message) {
+		System.out.println("Exit Controller : " + message);
+	}
 
 	@Override
 	public void ticketInserted(String ticketStr) {
-		// TODO Auto-generated method stub
```
Comment (engrhxn) [inline src/bcccp/carpark/exit/ExitController.java:38]: please remove this line.

**Review by engrhxn — COMMENTED** [review]
Please have a look in the code, I have put comments where it needs some changes.  Thank you.

File: src/bcccp/carpark/exit/ExitController.java (hunk 2 above)
Comment (iota333) [reply src/bcccp/carpark/exit/ExitController.java:38]: ok

**Review by engrhxn — APPROVED** [review]
Accepted. Thank you.

**Pull request merged**